cam_fleft_roi:
#  for 120deg
#  x_y_w_h: [1000, 0, 600, 1200 ]
#  for 140deg
  x_y_w_h: [ 900, 0, 700, 1200 ]

cam_fright_roi:
#  for 120deg
#  x_y_w_h: [0, 0, 600, 1200 ]
#  for 140deg
  x_y_w_h: [ 0, 0, 700, 1200 ]
